source/edge-server/connect.txt

```diff
@@ -170,58 +203,122 @@ modifications you can make to the :github:`Swift template app
    .. step:: Run the App
 
       Now you can build and run the app. When you log in, the app connects 
-      to the Edge Server. When you create new Items, those sync with the 
-      Edge Server. The Edge Server then syncs with the Atlas App Services App.
+      to the Edge Server instance. When you create new Items, those sync with 
+      the instance. The instance then syncs with the Atlas App Services App.
 
-      You can check the status of the Edge Server from your host.
+      You can check the status of the Edge Server instance from your host.
       With the client running, you can see ``"num_local_clients": 1`` in 
-      the Edge Server status. You can also see error messages reflected
+      the instance status. You can also see error messages reflected
       in the sync status.
 
-      After the next time the Edge Server syncs with your App Services App,
-      you can see synced changes reflected in the linked Atlas collection.
+      After the next time the Edge Server instance syncs with your App Services 
+      App, you can see synced changes reflected in the linked Atlas collection.
 
       .. important::
 
          If you have previously connected directly from the client to your 
          App Services App with Device Sync, you may need to clear state on the 
          simulator or device. Resetting your cache enables your client to 
-         connect to the Edge Server.
+         connect to the Edge Server instance.
 
          In a console app, delete the ``mongodb-realm`` directory. If you're
          using an iOS or Android simulator or device, uninstall and reinstall
          the app.
 
 .. _edge-server-wireprotocol-connect:
 
-Connect to the Edge Server with a MongoDB URI
----------------------------------------------
+Connect to the Edge Server with MongoDB Drivers and Tools
+---------------------------------------------------------
 
 You can connect to the Edge Server using standard MongoDB Drivers and tools.
 Clients use a specialized MongoDB URI connection string to connect to 
 Edge Server and send requests.
 
-The wire protocol server is on by default. You can connect to it via the 
-host machine using this MongoDB URI:
+The Wire Protocol server accepts these incoming connections, and is on by 
+default. You can connect to it using the host machine using this MongoDB URI:
 
 .. code:: console
 
-   mongodb://EDGE_SERVER_HOSTNAME:27021
+   mongodb://<username>:<password>@<hostname>:<port>?authMechanism=PLAIN&appName=::<auth-type>
+
+.. list-table::
+   :header-rows: 1
+
+   * - Element
+     - Description
+
+   * - ``<username>``
```

Comment:
note that email password users dont actually have to be email usernames, and indeed, this would complicate things because of the "@" having to be escaped. so I'd prefer guiding users toward non-email usernames

Reply:
or, we should give an example with the @ properly escaped

Reply:
Ok, I'm added an example using percent encoding to escape the `@`. If there's another way we'd prefer to show this, I'm happy to update.